src/app/todo-detail-modal/todo-detail-modal.component.ts

```diff
@@ -43,6 +43,10 @@ export class TodoDetailModalComponent implements OnInit {
     }
   }
 
+  public delete() {
+    this.todoService.deleteTodo(this.todo.id).subscribe();
```

Comment:
Wie handelst du den Positivfall? Woher weißt du, dass es gelöscht wurde, und wie gehst du mit einem Fehler in der UI um?